RFC-001-BAM.md

```diff
@@ -166,127 +169,92 @@ Ids MAY be skipped but MUST be ascending.
-
-    A response with `OK00` indicates that the responding node understood the request and processed it successfully.
-
-###### Sender errors (SE00-SE99)
-
-If a node responds with a status code from the `SE` family, this means that it was unable to process this request due to its content (similar to the HTTP 400 family). Sending nodes SHOULD NOT resend such a request without modification.
-
-Looking at these responses, it might at first be confusing why they are listed in the `SE` category even though it is usually the receiving end that is unable to process the request because it doesn't meet certain requirements.
-The reason why it is still a `SenderError` is because the sender can potentially adjust the request and try again if it has this kind of error handling built in.
-The receiving end however probably needs an upgrade to a newer version so that it understands these requests.
-It is therefore the sender's "fault" that the two nodes could not successfully communicate.
-
-1. SE00 - Bad request
-
-    A response with the status code `SE00` indicates that the request sent by sender was malformed and could not be processed.
-This is the most generic error that can be returned by a node.
-If no other errors code fits as to why the request was faulty, this one can be returned.
-
-2. SE01 - Unsupported mandatory header
-
-    Whenever a node receives a header that it does not understand, it should return a response with the SE01 status code.
-Responses with the SE01 status code MUST include the header `unsupported_headers` with its value set to a list of mandatory headers it didn't understand.
-If a node does not understand multiple headers, it SHOULD list all of them.
-
-3. SE02 - Unknown request type
-
-    A response with the status code `SE02` indicates that the sender sent a request with a `type` that was not understood.
-
-###### Receiver errors (RE00-RE99)
-
-Responses from the `RE` family indicate that the receiving node encountered an error during process of the request.
-A sending node MAY infer that the request itself was fine but the receiving end failed to process it due to an internal error which the sending end MUST NOT be held responsible for.
-
-1. RE00 - Internal error
-
-    `RE00` indicates that the node encountered an internal, unexpected error during processing of the request.
+The structure of the `body` of a REQUEST is entirely up to the application protocol that is defined through the REQUEST `type`.
+Similar to HTTP, application protocols MAY include some kind of 'Content-Type' in the headers in order to describe the encoding of the body.
+When designing an application protocol, it is common to overcome the question of whether a specific piece of data should be encoded as a header or be represented in the body.
+The rule of thumb here is that implementations should be able to parse all headers orthogonally.
+Hence, the structure/possible values of one header SHOULD NOT depend on those of other headers.
```

Comment:
🤔 we kind of have that with ledgers and assets (kinda)

Reply:
Yeah I would remove the `/possible values` bit of this sentence.